### DISCUSSION TITLE
Presets rsgain

### DISCUSSION THREAD

**Discussion by RogerRichardChapman** [complexlogic/rsgain#139] [Q&A]
Hi,
Sorry if this request seems obvious but I am not a coder. I like rsgain which I use on my Android phone in both easy mode and custom mode, for both internal storage and also SD card as follows:
SD Card
rsgain easy /storage/F72A-1310/Android/data/com.termux/files
Internal Storage 
rsgain easy /storage/emulated/0/Transition

I have found the preset files in
 /data/data/com.termux/files/usr/share/rsgain/presets
When I try to use a preset using the command below, I get "no directory specified" message. Can anyone give me the correct format? 
rsgain easy -p ebur128/storage/F72A-1310/Android/data/com.termux/files/usr/share/rsgain/presets

**Comment by complexlogic** [comment]
You should be able to just use `-p ebur128`, then the path of where your music library is
```
rsgain easy -p ebur128 /path/to/library
```

**Comment by RogerRichardChapman** [comment]
That's great. Work's fine. Thanks for replying so quickly.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
